### ISSUE TITLE
update attributes

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/recipe-search-api#23]

**Comment by sanjeeb9853** [comment]
When adding course attribute to recipe, don't I have to create a new course model and create one to many or many-to-many relation between course and recipe. The same goes for cuisine, holiday and flavors.

**Comment by atherdon** [comment]
yeah, you right @sanjeeb9853  1-to-many relation (one recipe - few attributes(i.e. few courses))

**Comment by sanjeeb9853** [comment]
Hi @atherdon I updated the attributes and have created a pull request, it's at https://github.com/ChickenKyiv/recipe-search-api/pull/66.

**Comment by atherdon** [comment]
@sanjeeb9853 yeah, i saw it, but i this is not exactly what we needed here.
we faced a similar probem before, when we also added separated relations for diets, allergies, etc.
But we decide to simplify the database logic and instead of having 5 relations - we add an `attribute` field, and this field contain all the stuff.
so i think we should create an arrays, each of them will be related to one `field`, like saulty and then we'll add that arrays to our import script, that stored at `bin` folder.
Tell me if this is not clear - adn i'll explain more

**Comment by atherdon** [comment]
@sanjeeb9853 tell me what should be our next steps
